tests/ci/cdk/util/iam_policies.py

```diff
@@ -95,7 +95,15 @@ def code_build_batch_policy_in_json(project_ids):
                     "codebuild:RetryBuild"
                 ],
                 "Resource": resources
-            }
+            },{
+                "Effect": "Allow",
+                "Action": [
+                    "ecr:GetDownloadUrlForLayer",
+                    "ecr:GetAuthorizationToken",
+                    "ecr:BatchGetImage"
+                ],
+                "Resource": "*"
```

Comment:
do we know the registry ID? `ecr:GetAuthorizationToken` on a resource wildcard is pretty broad.